### DISCUSSION TITLE
Oxygen tank refill shelves should hold 1 or 2 tanks only

### DISCUSSION THREAD

**Discussion by IgRAzm** [FakeFishGames/Barotrauma#13036] [Ideas and suggestions]
Currently if the sub uses the default diving suit+shelf template, you will get way more backup oxygen time with the tanks than before the treacherous tides update. This unnecessarily lessens the threat of oxygen station getting damaged or being off from power blackout. IMO reducing tank capacity is an easy solution that won't be a big deal for majority of subs. Only needs a slight sprite change but I think making tanks look bigger there would make sense, mb turn the tank sideways.

**Comment by Hazmat616** [comment]
Yeah I normally just grab a dive suit and the canisters below it so 1 or 2 would make sense.

**Comment by Arctodites** [comment]
Agreed.  This becomes exponentially unbalanced when there are a high number of suit locker + shelf units on the sub, and even moreso when the sub is stocked with diving masks.  Add in looting from outposts, and the game's whole economy quickly breaks.

One thing I love about dive missions is that you need to plan ahead for oxygen tank consumption: bring too few tanks and you die; bring too many, and you might have to throw a few away to free-up space in your inventory for tasty loot.  This penalty then has knock-on consequences for provisioning the sub, and it helps to balance the game's economy.

Apropos diving masks: currently it is very easy to horde diving masks in cabinets all over the sub, which makes it unwieldy to take stock of how many masks (and O2 tanks) you actually have aboard, and it becomes anyone's guess in an emergency as to where those masks are actually located.  So it would be helpful if we had a dedicated diving mask holder (akin to the 'weapon holder'), which openly shows the diving mask and passively encourages players to store their masks in pre-designated locations that everyone can anticipate.  This could be especially helpful with respect to bots: no more guessing where the bots have hidden your diving mask.

**Reply by Saevon** [reply]
Diving mask holder and bots putting away diving masks to empty holders should be its own suggestion!
